recipe/meta.yaml

```diff
@@ -1,65 +1,227 @@
-    - spacy <2.4.0
-    - textblob
-    - umap-learn
-    - wordcloud
-    - yellowbrick >=1.0.1
-    - numba <0.55
-    - numpy
-    - packaging
+    - setuptools
+    - wheel
 
-test:
-  imports:
-    - pycaret
+outputs:
+  - name: {{ name }}
+    build:
+      skip: true  # [py<38]
+      script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+    requirements:
+      host:
+        - python
+        - numpy {{ numpy }}
+        - pip
+        - setuptools
+        - wheel
+      run:
+        - python
+        # Base
+        - ipywidgets >=7.6.5
+        - ipython >=5.5.0
+        - tqdm >=4.62.0
+        # Can't numpy >=1.24 because of np.float deprecation
+        - {{ pin_compatible('numpy') }}
+        - pandas >=1.3.0,<2.0.0
+        - jinja2 >=1.2
+        - scipy <2.0.0
+        - joblib >=1.2.0
+        - scikit-learn >=1.0
+        - pyod >=1.0.8
+        - imbalanced-learn >=0.8.1
+        - category_encoders >=2.4.0
+        - lightgbm >=3.0.0
+        - numba >=0.55.0
+        - requests >=2.27.1
+        - psutil >=5.9.0
+        - markupsafe >=2.0.1
+        - importlib_metadata >=4.12.0
+        - nbformat >=4.2.0
+        - pickle5  # [py<38]
+        - cloudpickle
+        - deprecation >=2.1.0
+        - python-xxhash
+        - wurlitzer  # [not win]
+        # Plotting
+        - matplotlib-base >=3.3.0
+        - scikit-plot >=0.3.7
+        - yellowbrick >=1.4
+        - plotly >=5.0.0
+        - kaleido-core >=0.2.1
+        - schemdraw ==0.15
+        - plotly-resampler >=0.8.3.1  # [py<311]
+        # Time-series
+        - statsmodels >=0.12.1
+        - sktime >=0.16.1,!=0.17.1,<0.17.2
+        - tbats >=1.1.3
+        - pmdarima >=1.8.0,!=1.8.1,<3.0.0
+    test:
+      imports:
+        - pycaret
+      requires:
+        - pytest >=7.0.1
+        - moto >=3.0.7
+        - fugue >=0.8.0
+        - dask-core
+        - dash
+      #  - pip
+      #commands:
+        # Disable pip check because of errors:
+        # pycaret 3.0.2 requires kaleido, which is not installed.
+        #- pip check
+
+  - name: {{ name }}-models
+    build:
+      skip: true  # [py<38]
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('pycaret', exact=True) }}
+        # Models
+        - xgboost >=1.1.0
+        - catboost >=0.23.2
+        - kmodes >=0.11.1
+        - mlxtend >=0.19.0
+        # TODO: enable statsforecast before uploading to the destination channel
+        #- statsforecast >=0.5.5
+        - scikit-learn-intelex >=2023.0.1  # [x86_64]
+    test:
+      imports:
+        - pycaret
+
+  - name: {{ name }}-parallel
+    build:
+      skip: true  # [py<38]
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('pycaret', exact=True) }}
+        # Parallel
+        - fugue >=0.8.0
+        - flask >=2.0.3
+    test:
+      imports:
+        - pycaret
+
+  # - name: pycaret-full
```

Comment:
I would not comment out such a huge part of recipe but add a one-liner comment saying pycaret-full, *-analysis, *-tuners and *-mlops were disabled because of missing dependencies (and then put the link to explain)

Reply:
oh, and I would straight up delete these sub-packages

Reply:
If I enable all sub-packages and skip building them then will be an error:
`conda.exceptions.UnavailableInvalidChannel: HTTP 404 UNAVAILABLE OR INVALID for channel upload <file:///upload>`
I'll check if this error persists

Reply:
Isn't there another way of skipping them? Perhaps with selectors?

Reply:
@boldorider4 Looks like the issue has been resolved. I've uncommented optional sub-packages but skipped building them (when we will have all dependencies it will be helpful to enable one, many, or all sub-packages).